tests/snapshot-test/node/snapshot.test.js

```diff
@@ -1,7 +1,7 @@
 const { getESLintConfig, optimizeRules } = require('../helper');
 
 test('should match ESLint configuration snapshot: node', async () => {
-  const filePath = 'dummy.tsx';
+  const filePath = './dummy.js';
```

Comment:
The name of the dummy file specified for snapshot-test was incorrect, so it was corrected.